lib/learn_to_rank/judgements_to_svm.rb

```diff
@@ -0,0 +1,34 @@
+module LearnToRank
+  class JudgementsToSvm
```

Comment:
It would be great if this linked to some documentation on the SVM format and what it's used for - is this a tensorflow-specific thing, or is it wider than that?